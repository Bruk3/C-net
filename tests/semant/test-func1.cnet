int main() {
    int a = 4;
    int b;
    b = 5;
    // int b = 5;
    a = 4;
    b = a + b;
    b += a;
    return  b;
  //   return a;
}