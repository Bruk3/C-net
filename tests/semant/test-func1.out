int main()
{
{
int a = (int : 0);
(int : a=(int : 4));
}
int b = (int : 0);
(int : b=(int : 5));
(int : a=(int : 4));
(int : b=(int : (int : a) + (int : b)));
(int : b=(int : (int : b) + (int : a)));
return (int : b);
}

